homebrew_challenge/src/components/GeoHeatMap/RegionTypeSelector.jsx

```diff
@@ -18,6 +18,8 @@ function RegionTypeSelector({ areaType, setAreaType }) {
     throw new Error("Unrecognised setAreaType: " + setAreaType);
   }
   return (
+    <fieldset>
+      <legend>Select Boundaries:</legend>
```

Comment:
Formatting/indentation is a bit off here

Reply:
fixed